code/__HELPERS/traits.dm

```diff
@@ -206,6 +206,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 //***** ITEM TRAITS *****//
 #define TRAIT_BUTCHERS_HUMANS "butchers_humans"
 #define TRAIT_CMAGGED "cmagged"
+/// Wires on this will have their titles randomized for those with SHOW_WIRES
+#define TRAIT_OBSCURED_WIRES "obscured_wires"
```

Comment:
probably wanna add this to the global trait list so admemes can select it

Reply:
Done